tests/Activities.UnitTests/Internal/ActivityMetricsSenderTests.cs

```diff
@@ -33,13 +98,19 @@ public void SendActivityMetric_ProduceMetrics()
 			contextMock.Setup(e => e.DeploymentSlice).Returns("002");
 			contextMock.Setup(e => e.IsCanary).Returns(true);
 			contextMock.Setup(e => e.IsPrivateDeployment).Returns(false);
-			IExecutionContext context= contextMock.Object;
+			IExecutionContext context = contextMock.Object;
```

Comment:
nit: having that the definition of `contextMock` spans multiple lines, I would add a line above the assignment of the `context` variable to increase readability.